intellij/ast/build.gradle

```diff
@@ -11,6 +11,12 @@ intellij {
     updateSinceUntilBuild = false
 }
 
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
I think ci may fail due to it still using java 11?

Reply:
@jbarr21 do you expect this to be an issue?

Reply:
you will also need to update github actions config to build this module w/ java 17, but no, it shouldnt affect prod apps

Reply:
Reading more through the [toolchain doc](https://docs.gradle.org/current/userguide/toolchains.html) - it seems that we may not need to touch the github workflow files because gradle handles that for us? (Probably also explains why this builds fine despite the workflow setup still runs on java 11)

Reply:
Yes. Toolchain will have Gradle download the specified JDK for building the project. Workflow JDK will still run the steps in the flow, but the project build will ultimately be done by the toolchain-specified JDK.